pkgs/development/ocaml-modules/wodan/irmin.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, buildDunePackage, irmin-chunk, irmin-git, irmin-unix
+, mirage-block-ramdisk, mirage-block-unix, wodan }:
+
+buildDunePackage rec {
+  pname = "wodan-irmin";
+  inherit (wodan) version src useDune2;
+
+  propagatedBuildInputs = [
+    irmin-chunk
```

Comment:
`io-page-unix` might be missing here.

Reply:
Builds anyway?

Reply:
@GrahamcOfBorg build ocamlPackages.wodan-irmin